README.md

```diff
@@ -18,7 +18,7 @@ For a visual representation of the project architecture and details on the desig
 
 ### Architecture Diagram
 
-![Architecture Diagram](https://github.com/fm1psy/c11-poseidon-earthquake-monitoring/blob/main/diagrams/earthquake_architecture.png)
+![Architecture Diagram](https://github.com/fm1psy/c11-poseidon-earthquake-monitoring/blob/main/diagrams/architecture-diagram.png)
```

Comment:
I noticed you changed the filename at the end of the image url. Is there a reason you replaced the _ with a dash?

Reply:
No real reason, I'll change it to '_' for consistency.